pkg/model/proto/model.proto

```diff
@@ -674,6 +694,54 @@ message DetachVolumeOpts {
     string context = 4;
 }
 
+// CreateFileShareOpts is a structure which indicates all required properties for creating a file share.
+message CreateFileShareOpts {
+    // The uuid of the file share, optional when creating.
+    string id = 1;
+    // The name of the file share, required.
+    string name = 2;
+    // The requested capacity of the file share, required.
+    int64 size = 3;
+    // The description of the file share, optional.
+    string description = 4;
+    // The locality that file share belongs to, required.
+    string availabilityZone = 6;
+    // The service level that file share belongs to, required.
+    string profileId = 7;
+    // The uuid of the pool on which file share will be created, required.
+    string poolId = 8;
+    // The name of the pool on which file share will be created, required.
+    string poolName = 9;
+    // The metadata of the file share, optional.
+    map<string, string> metadata = 10;
+    // The storage driver type.
+    string driverName = 11;
+    // The Context
+    string context = 12;
+    // The Serialized profile
+    string profile = 13;
+}
+
+// DeleteFileShareOpts is a structure which indicates all required properties
+// for deleting a file share.
+message DeleteFileShareOpts {
+    // The uuid of the fileshare, required.
+    string id = 1;
+    // The service level that fileshare belongs to, required.
+    // This item will be replace by profile
+    string profileId = 2;
```

Comment:
Remove it.  We already have profile below.

Reply:
Done

Reply:
Done